taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    context = {
+        "num_drivers": num_drivers,
```

Comment:
Refrain from creating variables. Instead, directly assign the result as values

` "num_drivers": Driver.objects.count()`